compiler-core/src/type_/environment.rs

```diff
@@ -606,13 +606,36 @@ impl<'a> Environment<'a> {
         }
     }
 
+    // List all possible imports values
+    pub fn imported_modules_values(&self) -> Vec<EcoString> {
+        self.imported_modules
+            .iter()
+            .map(|(module_name, (_, module))| {
+                module
+                    .values
+                    .keys()
+                    .map(|value| format!("{}.{}", module_name, value).into())
+                    .collect::<Vec<EcoString>>()
```

Comment:
```suggestion
                    .map(move |value| format!("{}.{}", module_name, value).into())
```

Regarding the ci-lint error about `flat_map`, you can additionally avoid the intermediate `.collect()`s by moving the `module_name` reference into the closure. Its lifetime should last fine for the duration of this whole function (or otherwise the borrow checked would yell).

Reply:
Could use that cool eco_format macro too

Reply:
Thanks for the feedback, appreciated. I will update this 🙂

Reply:
I think you missed the main part of my suggestion: removing the `.collect::<Vec<EcoString>>()`. Adding the `move` just enables that :)

Reply:
Updated it, Looks good to you? Thanks :)